### PR TITLE
Add Italian translation.

### LINKED ISSUE #48: Add Italian.
The Italian translation of Disk Detective is available now; add an Italian file to fix generic tags translation.

screen shot 2016-03-09 at 14 02 26

### PR DESCRIPTION
**Pull request by simoneduca**
Closes #48

### CODE REVIEW

File: translations/it.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    password: 'Password',
+    email: 'Email',
+    realName: 'Nome reale',
+    whyRealName: 'Questo verra\' utilizzato per ringraziare i collaboratori, ad esempio, in presentazioni e posters. Se non vuoi che il tuo nome venga utilizzato in pubblico, lascia questo spazio vuoto.', 
+    noAccount: 'Non hai un profilo?',
+    agreeToPrivacyPolicy: 'Accetto <a href="https://www.zooniverse.org/privacy" target="_blank">le norme sulla Privacy</a>.',
+    betaPreference: 'Vorrei ricevere notifiche per testare e commentare su progetti Zooniverse non ancora pubblicati.',
+    forgotPassword: 'Hai dimenticato la tua password?',
+    badLogin: 'Username or password incorretti',
+    signInFailed: 'Sign in failed.',
```
Comment (eatyourgreens) [inline translations/it.js]: Should this still be in English?

**Comment by eatyourgreens** [comment]
LGTM, apart from the 'Sign in failed' bit.

File: translations/it.js (hunk 1 above)
Comment (simoneduca) [inline translations/it.js]: Doh!

**Comment by eatyourgreens** [comment]
:+1: Thanks!

**Pull request merged**